Homework3/Fizz_buzz.py

```diff
@@ -0,0 +1,20 @@
+# FizzBuzz
+# Напишите программу, которая печатает цифры от 1 до 100,
+# но вместо чисел, кратных 3 пишет Fizz, вместо чисел
+# кратный 5 пишет Buzz, а вместо чисел одновременно
+# кратных и 3 и 5 - FizzBuzz
+
+line_count = 0
+for i in range(1, 101):
+    current = ''
+    if i % 3 == 0:
+        current = 'Fizz'
+    if i % 5 == 0:
+        current += 'Buzz'
+    if current == '':
+        current = str(i)
```

Comment:
we don't need str if we want just to print it

Reply:
ok